maps/tether/tether-01-surface1.dmm

```diff
@@ -13949,7 +13949,8 @@
 /area/rnd/external)
 "aHg" = (
 /obj/machinery/camera/network/research_outpost{
-	dir = 4
+	dir = 4;
+	network = list("Research")
```

Comment:
Correct fix here is to change the *type* to `/obj/machinery/camera/network/research` instead of overriding the network.

Reply:
I was actually unaware of that. Thank you.

Reply:
No problem, except you forgot to actually remove the network var on this one.  It was removed on the one below.

Reply:
No problem, except you forgot to actually remove the network var on this one. It was removed on the one below.